docs/content/v0-upgrade.md

```diff
@@ -470,6 +470,25 @@ OPA binary of version 1.0 or later.
 If you run into any issues while upgrading a Rego project, please drop a message
 in the #help channel on the [OPA Slack](https://slack.openpolicyagent.org/).
 
+## Upgrading OPA Instances
+
+Prior to OPA 1.0, when running in server mode (`opa run --server/-s`), OPA would
+bind to all interfaces by default. In OPA 1.0,
+[OPA will bind to `localhost`](https://github.com/open-policy-agent/opa/issues/6286)
+by default instead. Though not inherently insecure in a trusted environment,
+it's good practice to bind OPA to localhost by default if OPA is not intended to
+be exposed to remote services.
+
+If you need to replicate the v0.x behaviour, you can use the `--addr` flag to
```

Comment:
Perhaps worth explicitly calling out that this is likely necessary when running OPA in Docker?